ADOPTERS.md

```diff
@@ -17,6 +17,21 @@ teams can deploy reliably, prevent failure and assure the customer experience.
 We use Antrea's Open vSwitch support to tune how services interact in Kubernetes
 clusters. We are @glasnostic on Twitter.
 
+**[Transwarp](https://www.transwarp.cn)**
+
+Transwarp is committed to building enterprise-level big data infrastructure
+software, providing enterprises with infrastructure software and supporting
+around the whole data lifecycle to build a data world of the future.
+1. We use Antrea's AntreaClusterNetworkPolicy and AntreaNetworkPolicy to protect
+big data software for every tenant of our kubernetes platform.
+2. We use Antrea's Open vSwitch to support Pod-To-Pod network between flannel and
+antrea clusters, and also between antrea clusters
+3. We use Antrea's Open vSwitch to support Pod-To-Pod network between flannel and
+antrea nodes in one cluster for upgrading.
+4. We use Antrea's Egress feature to keep the original source ip to ensure
+Internal Pods can get the real source IP of the request.
+You can contact us with mkt@transwarp.io
```

Comment:
This needs to start a newline,  otherwise it would appear as a sentence of the 4th item.

Reply:
Thanks for your review, modified